odl/solvers/advanced/proximal_operators.py

```diff
@@ -1079,6 +1080,91 @@ def _call(self, x, out):
     return ProximalCConjKL
 
 
+def proximal_cconj_kl_version_two(space, lam=1, g=None):
```

Comment:
The other one is only default in data-matching term... this version has been used as regularizing functional in different settings under the name maximum entropy or cross entropy:

- http://ieeexplore.ieee.org/document/1056144/?arnumber=1056144&tag=1
- http://mnras.oxfordjournals.org/content/211/1/111.short
- http://ieeexplore.ieee.org/document/1456693/?arnumber=1456693
- https://www.osapublishing.org/josa/abstract.cfm?uri=josa-62-4-511
- http://ieeexplore.ieee.org/document/771159/?arnumber=771159
- http://iopscience.iop.org/article/10.1088/0266-5611/7/6/004/meta

and this is only the tip of the iceberg.

With that said, we can still settle for keeping the name of the other and giving a new name to this one. `cross_entorpy`?